compiler/src/iree/compiler/Codegen/LLVMGPU/LLVMGPUTensorCoreVectorization.cpp

```diff
@@ -119,10 +120,132 @@ static Optional<SmallVector<int64_t>> getGPUTCNativeVectorSize(Operation *op) {
   return std::nullopt;
 }
 
+static int getVectorContractOpOperandId(
+  vector::ContractionOp contractOp, Operation *op) {
+  if (contractOp.getLhs() == op->getResult(0)) return 0;
```

Comment:
It looks odd to do op->getResult(0), you should probably pass directly the `Value` to the function